### DISCUSSION TITLE
Path of Exile

### DISCUSSION THREAD

**Discussion by mrleblanc101** [elrumo/macOS_Big_Sur_icons_replacements#1198] [Icon request]
Hi, 
Could someone make a PoE icon for Big Sur please ?

**Comment by cake03** [comment]
Could you please upload the original icon?

**Reply by mrleblanc101** [reply]
[shortcut.icns.zip](https://github.com/elrumo/macOS_Big_Sur_icons_replacements/files/6620454/shortcut.icns.zip)

**Reply by cake03** [reply]
@mrleblanc101, I've uploaded the icon, please close the issue :)

**Comment by cake03** [comment]
![Icon](https://user-images.githubusercontent.com/80169337/121540758-5e05bd80-ca24-11eb-8f67-227c3755114b.png)
# [[Download]](https://user-images.githubusercontent.com/80169337/121540758-5e05bd80-ca24-11eb-8f67-227c3755114b.png)
